apps/native-component-list/src/screens/WebBrowserScreen.tsx

```diff
@@ -89,8 +99,13 @@ export default class WebBrowserScreen extends React.Component<{}, State> {
       browserPackage: this.state.selectedPackage,
       enableBarCollapsing: this.state.barCollapsing,
       showInRecents: this.state.showInRecents,
+      readerMode: this.state.readerMode,
+      enableDefaultShare: this.state.enableDefaultShare,
     };
-    const result = await WebBrowser.openBrowserAsync(url, args);
+    const result = await WebBrowser.openBrowserAsync(
+      'https://proandroiddev.com/say-no-to-baseactivity-and-basefragment-83b156ed8998?source=grid_home----c72404660798-----0-1-----------------18---39b94dfe_9ccf_4e8c_803f_bcf970042e56--',
```

Comment:
I appreciate a good read, but I'm not sure we should start opening a Medium post while testing `WebBrowser` implementation.